app/views/admin/events/index.pdf.prawn

```diff
@@ -0,0 +1,30 @@
+prawn_document(force_download: true, filename: @pdf_filename, page_layout: :landscape) do |pdf|
+  table_array = []
+  header_array = ['Title',
+                  'Abstract',
+                  'Start time',
+                  'Submitter',
+                  'Speaker',
+                  'Event Type',
+                  'Track',
+                  'Difficulty Level',
+                  'State']
+
+  table_array << header_array
+  @events.each do |event|
+    row = []
+    row << event.title
+    row << event.abstract
+    row << event.start_time.to_s
+    row << (event.submitter.present? ? event.submitter.name.to_s : "")
```

Comment:
For styling purposes those quotes can be single quotes instead of double quotes.